src/i2c.rs

```diff
@@ -25,6 +25,8 @@ pub enum Error {
     #[doc(hidden)] _Extensible,
 }
 
+// AJM - Should these pin mappings be identical to the 30x crate?
```

Comment:
I left in all the files I hadn't implemented, but they're commented out in lib.rs ;)